server/models/ClusterModel.ts

```diff
@@ -57,7 +57,7 @@ class ClusterModel {
       }
     
     res.locals.elements = elements
-    console.log(elements)
+    //console.log(elements)
```

Comment:
Is this comment intended for future dev? If so, might be worth calling out why it's useful enough to leave in the codebase going forward.